packages/client-ts/src/client.ts

```diff
@@ -48,6 +49,8 @@ class Client {
       : {};
     const config = _.merge(defaultConfig, userConfig);
 
+    this.defaultVaultAppName = DEFAULT_CONFIG.vaultAppName;
```

Comment:
Should come from `config`, otherwise user specified defaults won't be applied.

Reply:
the `config` does not have `defaultVaultAppName`

```javaScript
export interface ClientConfig {
	/**
	 * Environment to load by default.
	 *
	 * Environment configuration can still be overriden by config items.
	 */
	environment?: EnvironmentType;

	/**
	 * URL of Verida DID Server node to use.
	 */
	didServerUrl?: string;

	/**
	 * Specify custom schema paths (typicaly for local development).
	 */
	schemaPaths?: object;
}

```
Not sure  if its possible to read it from  `config`